spec/jobs/scraping_process_distributing_job_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ScrapingProcessDistributingJob, type: :job do
+  include ActiveJob::TestHelper
+  ActiveJob::Base.queue_adapter = :test
+
+  describe '#perform' do
+    context 'given valid inserted keywords' do
```

Comment:
how about the invalid case?

Reply:
Fixed in e59d041b1e08eed554ed1b3f00249c7f078f1a35